app/views/admin/api_connections/index.html.erb

```diff
@@ -0,0 +1,75 @@
+<div class="w-full p-4 flex-col justify-start items-start gap-4 inline-flex">
+  <div class="self-stretch bg-white rounded-md border border-gray-200 flex-col justify-start items-start flex">
+    <div class="self-stretch p-6 border-b border-gray-200 justify-start items-center gap-4 inline-flex">
+      <div class="grow shrink basis-0 text-gray-900 text-xl font-semibold leading-[35px]">ÚPVS prepojenia</div>
+    </div>
+    <div class="self-stretch flex-col justify-start items-start flex">
+      <% @api_connections.select {|a| a.upvs_type?}.each do |api_connection| %>
```

Comment:
Ak by sme si nechceli tahat do core domeny tieto veci, tak mozeme spravit nejaky group_by(&:type) a potom to nechat na polymorfiu (ta funguje aj pri partials), ale necham na teba ci sa s tym chceme babrat.

Reply:
Teraz sme v celkom casovej tiesni, skusila by som si rada, ale asi v neskorsom kole uprav. Nech teraz nasadime cim skor.